app/api/tickets.py

```diff
@@ -20,6 +20,13 @@
 from app.api.helpers.exceptions import ConflictException, MethodNotAllowed, UnprocessableEntity
 from app.api.helpers.db import get_count
 
+
+def validate_ticket_price(ticket_type, price):
+    if ticket_type != 'free' and price <= 0:
```

Comment:
This check is somewhat buggy. It'll pass it ticket type is free and price < 0

Reply:
Yeah @shreyanshdwivedi.  Removed it.  please have a look now.